tests/screenshot/apps/components/ContextualPoppDecorator.js

```diff
@@ -0,0 +1,10 @@
+import ContextualPopupDecorator from '../../../../ContextualPopupDecorator';
+import React from 'react';
+
+const ContextualPopupDecoratorTests = [
+	<ContextualPopupDecorator />,
+	<ContextualPopupDecorator direction="above" />,
+	<ContextualPopupDecorator noArrow />
+];
+
+export default ContextualPopupDecoratorTests;
```

Comment:
Please check this file name. Not ContextualPoppDecorator but ContextualPopupDecorator.js.